docs/sampling/README.md

```diff
@@ -0,0 +1,282 @@
+value and a Randomness value, each containing 56 bits of information.
+
+A constant known as _maximum adjusted count_ (`MaxAdjustedCount`),
+with value `0x100000000000000`, (which can also be expressed
+`0x1p+56`, `math.Pow(2, 56)`, or `math.Ldexp(1, 56)`), defines the
+exclusive upper limit of these values.
+
+Logically, both Threshold (`T`) and Randomness (`R`) are represented
+as unsigned integers in the range `0` through `0xffffffffffffff` or
+`MaxAdjustedCount - 1`.  Items of telemetry are selected (i.e.,
+"sampled") when their Threshold value is less than or equal to their
+Randomness value, or `T <= R`.
+
+Sampling probability is defined by the following expression:
+
+```
+Probability = (MaxAdjustedCount - Threshold) / MaxAdjustedCount
+```
+
+In a Span-to-Metrics or Logs-to-Metrics pipeline, each item of
+telemetry is representative of an _adjusted count_ number of items in
+the original population.  Adjusted count is the inverse of sampling
+probability, and `MaxAdjustedCount` (defined above) is the inverse of
+the smallest supported sampling probability (which can also be
+represented as `0x1p-56`, `math.Pow(2, -56)`, or `math.Ldexp(1,
+-56)`).
+
+For the tracing signal, Threshold and Randomness propagate via W3C
+Trace Context `tracestate`.  When they appear in the `tracestate`, the
+Threshold and Randomness properties are called "T-value" and
+"R-value"; they are represented in the OpenTelemetry section of the
+`tracestate` (having vendor tag `ot`), using properties named `th` and
+`rv`, respectively.
+
+For the logs signal, which generally does not record the W3C Trace
+Context `tracestate`, sampling attributes are meant to be expressed
+using log record attributes with the same definition as T-value and
+R-value.
+
+For more information on how to perform and interpret probability
+sampling based on these properties, [consult OTEP 235][OTEP235].
+
+### Overriding sampling decisions
+
+Instrumentation authors and end-users that wish to prioritize an item
+of telemetry for collection in spite of sampling can add a
+`sampling.priority` attribute.  This attribute is a suggestion, a way
+of recognizing the importance of a certain event and requesting
+additional consideration from the collection pipeline.  For example, a
+user could write this code snippet:
+
+```
+   if err := doSomething(); err != nil {
+     if err == VERY_SERIOUS {
+       span.SetAttribute("sampling.priority", 10000)
+     }
+     return err
+   }
+```
+
+Samplers and sampling processors SHOULD pass items of telemetry to the
+exporter, independent of their default sampling mechanism, when the
+`sampling.priority` attribute is present and non-zero.  Although the
+priority is an integer, mathematical weight is not prescribed and no
+other specific behaviors are required.
+
+### Overriding sampling randomness
+
+Sampling system designers are able to override sampling randomness on
+an item-by-item basis, which may be done for several reasons, including
```

Comment:
nit: It seems item-by-item basis is only available for logs since tracestate will likely to be propagated to children and out-of-process.

Reply:
Agreed -- see a55667f.